apps/mobile/src/screens/NftDetailScreen/NftAdditionalDetailsTezos.tsx

```diff
@@ -60,19 +62,19 @@ export function NftAdditionalDetailsTezos({
 
   return (
     <View className="flex flex-col space-y-4">
-      {token.contract?.creatorAddress?.address && (
-        <DetailSection>
-          <DetailLabelText>CREATED BY</DetailLabelText>
+      <>
+        {token.contract?.creatorAddress?.address && (
+          <DetailSection>
+            <DetailLabelText>CREATED BY</DetailLabelText>
 
-          {/* TODO(Terence) When the contract screen is ready, setup the onPress here */}
-          <LinkableAddress
-            chainAddressRef={token.contract.creatorAddress}
-            type="NFT Detail Creator Address"
-          />
-        </DetailSection>
-      )}
+            {/* TODO(Terence) When the contract screen is ready, setup the onPress here */}
```

Comment:
is this comment still valid? looks like `LinkableAddress` might handle pressing on the address